aws/resource_aws_spot_instance_request.go

```diff
@@ -123,6 +136,25 @@ func resourceAwsSpotInstanceRequest() *schema.Resource {
 			}
 			return s
 		}(),
+
+		CustomizeDiff: customdiff.All(
+			SetTagsDiff,
```

Comment:
Good catch with the `SetTagsDiff`. Dynamically constructed `Schema` from `resourceAwsInstance().Schema` with additions = 😭.